packages/communication-react/stable-review/communication-react.api.md

```diff
@@ -1363,6 +1630,14 @@ export type LocalizationProviderProps = {
     children: React_2.ReactNode;
 };
 
+// @public (undocumented)
```

Comment:
This shouldn't be part of the public API

@dmceachernmsft

Reply:
@prprabhu-ms This is going out at a different time than CallWithChat? that because of it showing up in the calling composite?